config/v1/techpreview.infrastructure.testsuite.yaml

```diff
@@ -528,37 +528,149 @@ tests:
         platform: GCP
         platformStatus:
           type: GCP
-          gcp:
-            clusterHostedDNS: "Enabled"
+          gcp: {}
     updated: |
       apiVersion: config.openshift.io/v1
       kind: Infrastructure
       spec: {}
       status:
+        controlPlaneTopology: "HighlyAvailable"
+        infrastructureTopology: "HighlyAvailable"
         platform: GCP
         platformStatus:
           type: GCP
+          gcp: {}
+    expected: |
+      apiVersion: config.openshift.io/v1
+      kind: Infrastructure
+      spec: {}
+      status:
+        cpuPartitioning: "None"
+        infrastructureTopology: "HighlyAvailable"
+        platform: GCP
+        platformStatus:
           gcp:
-            clusterHostedDNS: "Disabled"
-    expectedStatusError: "status.platformStatus.gcp.clusterHostedDNS: Invalid value: \"string\": clusterHostedDNS is immutable and may only be configured during installation"
-  - name: Should not be able to remove GCP cluster hosted DNS from platformStatus.gcp
+            cloudLoadBalancerConfig:
+              dnsType: "PlatformDefault"
+          type: GCP
+        controlPlaneTopology: "HighlyAvailable"
+  - name: Should not allow changing the immutable dnsType field
     initial: |
       apiVersion: config.openshift.io/v1
       kind: Infrastructure
-      spec: {}
+      spec:
+        platformSpec:
+          gcp: {}
+          type: GCP
       status:
+        controlPlaneTopology: HighlyAvailable
+        infrastructureTopology: HighlyAvailable
         platform: GCP
         platformStatus:
+          gcp:
+            cloudLoadBalancerConfig:
+              dnsType: ClusterHosted
```

Comment:
You wanted to have a test for these two lines with `clusterHosted` omitted right? I don't see that test presently